__unported__/purchase_payment/__openerp__.py

```diff
@@ -23,6 +23,7 @@
     "version" : "1.0",
     "author" : "Pexego",
     "website": "www.pexego.es",
+    "license": "AGPL-3",
     "license" : "GPL-3",
```

Comment:
Double key here. Also note that you need permission to relicense. Having a GPL module in the OCA is not problematic anyway.